ml4gw/transforms/qtransform.py

```diff
@@ -130,15 +127,10 @@ def forward(self, fseries: torch.Tensor, norm: str = "median"):
         if norm:
             norm = norm.lower() if isinstance(norm, str) else norm
             if norm == "median":
-                # I swear there's a better way to do this
-                medians = torch.quantile(energy, q=0.5, dim=-1)
-                medians = medians.repeat(energy.shape[-1], 1, 1)
-                medians = medians.transpose(0, -1).transpose(0, 1)
+                medians = torch.quantile(energy, q=0.5, dim=-1, keepdim=True)
```

Comment:
Any reason not to use `torch.median`?

Reply:
Yeah, `torch.median` has different behavior than `numpy.median` for even-length arrays. It always takes the lower value, rather than the average of the two middle values.